cli.sh

```diff
@@ -97,19 +97,28 @@ set_main_variables() {
 #####################################################
 # Checks if the current directory is
 #   a Git repository or not.
-#
+
 # Returns:
 #   0 if it is likely a Git repository,
 #   1 otherwise
 #####################################################
 is_running_in_git_repo_root() {
-    if ! git status >/dev/null 2>&1; then
-        return 1
-    fi
-
+    git rev-parse >/dev/null 2>&1 || return 1
     [ -d "${CURRENT_GIT_DIR}" ] || return 1
```

Comment:
Please double-check that this still makes sense for bare repos.
The `${CURRENT_GIT_DIR}` variable is set to `git rev-parse --common-dir` or similar, no idea if that returns  something sensible for bare repos. /final